### PR TITLE
Add Scevan R package recipe

### PR DESCRIPTION
**Pull request by andreiprodan**
Recipe for GitHub Scevan R package

### CODE REVIEW

File: recipes/scevan/meta.yaml
Comment (martin-g) [inline recipes/scevan/meta.yaml:19]: I think this should be `host`, not `build`.
R, as Python, is an interpreted language, so these dependencies should be listed in `requirements>host`

File: recipes/scevan/meta.yaml
Comment (martin-g) [inline recipes/scevan/meta.yaml:48]: Hm!
I think is not correct!

1) This will install the latest code from the default branch. I think you need to specify the tag, e.g. `, ref = "1.0.0"`.
2) Why do you install it with devtools from Github at all ? IMO you should test the locally installed library instead

File: recipes/scevan/meta.yaml
Comment (andreiprodan) [reply recipes/scevan/meta.yaml:19]: will edit, thanks!

File: recipes/scevan/meta.yaml
Comment (andreiprodan) [reply recipes/scevan/meta.yaml:48]: will add the tag!

File: recipes/scevan/meta.yaml
Comment (martin-g) [inline recipes/scevan/meta.yaml:33]: Why this dependency is left to be (pre-)installed by the user ?
I think it should be a proper runtime dependency and lines https://github.com/bioconda/bioconda-recipes/pull/45649/files#diff-180caf3dc4ce15be03b03267239715f1249fa8f1d7e031a38dc7624e5189ef07R46-R48 / https://github.com/bioconda/bioconda-recipes/pull/45649/files#diff-180caf3dc4ce15be03b03267239715f1249fa8f1d7e031a38dc7624e5189ef07R27 could be removed

Comment (martin-g) [reply recipes/scevan/meta.yaml:33]: You may need a build script like https://github.com/bioconda/bioconda-recipes/pull/45645/files#diff-0d601fcf99e217658acf5443fb7fb9fe3d9691742cee85bccdb60bcfbdb200acR2 to install the library locally and test it without `devtools`

Comment (andreiprodan) [reply recipes/scevan/meta.yaml:33]: could not a way to install "yaGST" other than with `devtools::install_github()`. "yaGST" is not on CRAN or Bioconductor or conda

Comment (martin-g) [reply recipes/scevan/meta.yaml:33]: I see!
Maybe a recipe should be added for `yaGST` first ...
I'll let the Bioconda maintainers to decide what would be the best!

Comment (pcm32) [reply recipes/scevan/meta.yaml:33]: I would suggest that this gets shipped as it is and eventually when someone adds that additional recipe, then it could be added as conda dependency here? Otherwise it is harsh on contributors to have to chase every single dependent package. There are other packages that do installations of deps via devtools.

**Comment by pcm32** [comment]
With the above artifact @andreiprodan you could test locally if the code on the SCEVAN Readme works well.

File: recipes/scevan/meta.yaml
Comment (pcm32) [inline recipes/scevan/meta.yaml:2]: I think that the package name should have `r-` appended.
```suggestion
  name: r-scevan
```

File: recipes/scevan/meta.yaml
Comment (pcm32) [reply recipes/scevan/meta.yaml]: This tag is fairly old though, around a year. I wonder if we can ask the maintainer to make a new release.

Comment (andreiprodan) [reply recipes/scevan/meta.yaml]: updated to 1.0.1

**Comment by pcm32** [comment]
If the bot doesn't give the artifacts, they can be normally located through CI runs, for instance previous one to your last commit is here: https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=50666&view=artifacts&pathAsName=false&type=publishedArtifacts

**Comment by pcm32** [comment]
@BiocondaBot please fetch artifacts

File: recipes/r-scevan/meta.yaml
Comment (bgruening) [inline recipes/r-scevan/meta.yaml:20]:
```suggestion
```

File: recipes/r-scevan/meta.yaml
Comment (bgruening) [inline recipes/r-scevan/meta.yaml:25]:
```suggestion
```

File: recipes/r-scevan/meta.yaml
Comment (bgruening) [inline recipes/r-scevan/meta.yaml:53]: please use SPDX identifiers

Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml:51]: updated license to 'GPL-3.0-only' identifier

**Comment by andreiprodan** [comment]
@BiocondaBot please fetch artifacts

**Comment by cbrueffer** [comment]
Please remove the `LICENSE` file, you can point to the one installed by `r-base` instead: `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'`

**Review by cbrueffer — CHANGES_REQUESTED** [review]

File: recipes/r-scevan/meta.yaml
Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml]: comments have been removed

File: recipes/r-scevan/meta.yaml
Comment (pcm32) [inline recipes/r-scevan/meta.yaml:16]: This is now missing `miccec/yaGST` probably.

File: recipes/r-scevan/meta.yaml
Comment (pcm32) [inline recipes/r-scevan/meta.yaml:46]: I suspect that these shouldn't be in the tests... yaGST needs to be installed in the script part.

**Comment by pcm32** [comment]
For some reason the build is again re-installing the dependencies from R when doing R BUILD of the SCEVAN, I remember there is a way of telling R to build ignoring dependencies, as in our case they are all installed either by devtools or by conda.

File: recipes/r-scevan/meta.yaml
Comment (pcm32) [inline recipes/r-scevan/meta.yaml:39]: Ok, the issue I think is that all of this are missing in host AFAIK, that is why is trying to install again all these dependencies.

Comment (pcm32) [reply recipes/r-scevan/meta.yaml:39]: putting these in `host:` means that when it reaches the R CMD INSTALL --build for SCEVAN, then it won't try again to install them. Sorry, I hadn't paid attention to `run` and `host` sections. They also need to be present in run of course.

File: recipes/r-scevan/meta.yaml
Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml]: done (in a separate build.sh script)

File: recipes/r-scevan/meta.yaml
Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml]: added miccec/yaGST installation via devtools to build.sh script

File: recipes/r-scevan/meta.yaml
Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml]: removed "install_github('AntonioDeFalco/SCEVAN')"; now installed with "R CMD INSTALL --build . "

File: recipes/r-scevan/meta.yaml
Comment (andreiprodan) [reply recipes/r-scevan/meta.yaml:53]: fixed now

**Comment by pcm32** [comment]
Let me try the artifact.

**Comment by pcm32** [comment]
@BiocondaBot please fetch artifacts

**Comment by pcm32** [comment]
Great work @andreiprodan , and thanks for the perseverance!

- Artifact installs correctly
- This [example](https://htmlpreview.github.io/?https://github.com/AntonioDeFalco/SCEVAN/blob/main/vignettes/IntratumoralHeterogeneityInGlioblastoma.html) runs to completion, without errors and the figures generated look the same as in the example.

On my side I'm happy with this as it is.

**Review by pcm32 — APPROVED** [review]
LGTM and I have tested the artifact to work with real examples.

**Comment by pcm32** [comment]
@BiocondaBot please add label

**Comment by pcm32** [comment]
@cbrueffer I think that your requests have been addressed.

**Review by cbrueffer — APPROVED** [review]

**Comment by cbrueffer** [comment]
Nice job everyone!

**Pull request merged**
